components/style/values/computed/mod.rs

```diff
@@ -41,6 +44,13 @@ pub trait ToComputedValue {
 
     #[inline]
     fn to_computed_value(&self, _context: &Context) -> Self::ComputedValue;
+
+    #[inline]
+    /// Convert a computed value to specified value form.
+    ///
+    /// This will be used for recascadeing during animation.
+    /// Such to_specified_valued values should recompute to the same value.
+    fn to_specified_value(computed: &Self::ComputedValue, context: &UncomputeContext) -> Self;
```

Comment:
A more general comment/question: is there an advantage to putting this function on `ToComputedValue` rather than on a new trait that the specified value types implement?  Is it just so that we can have two function definitions side by side, rather than potentially in separate modules/files?